dev-app/src/screens/CollectCardPaymentScreen.tsx

```diff
@@ -226,7 +262,7 @@ export default function CollectCardPaymentScreen() {
       return;
     }
 
-    if (!paymentIntent?.id) {
+    if (!paymentIntent) {
```

Comment:
Out of curiosity, when would payment intent be null and `paymentIntentError` be null? I wonder if we can remove this if block.

Reply:
Resolving as this was addressed.